lang.cpp

```diff
@@ -59,26 +59,23 @@ int main(){
 */
 
   vector<string> transfer;
```

Comment:
As I said, you need a vector of vectors. The code is better divided into the vector of commands. And a command on the vector of tokens